string[] ssids
string[] macs
float32[] strengths